taxonomies/ingredients.txt

```diff
@@ -31159,9 +31162,18 @@ it:addensante gomma arabica
 #
 ##################################################################################
 
+#comment:en:Is used as a generic descriptor for (natural) sugar
+en:added sugar, saccharide
```

Comment:
saccharide is a synonym of carbohydrates, it should not be a synonym of "added sugars". We only want monosaccharides and disaccharides in en:added-sugar